cmd/kubeadm/app/preflight/checks_test.go

```diff
@@ -26,7 +26,6 @@ import (
 
 	"os"
 
-	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
```

Comment:
Whoa, I thought including the same package twice was invalid, seems like golang allowed it anyway... :)